R/build_multinom.R

```diff
@@ -32,13 +32,25 @@ build_multinom <- function(data, formula, ...){
 augment.multinom <- function(model, data = NULL, newdata = NULL) {
 
   predicted_label_col <- avoid_conflict(colnames(data), "predicted_label")
+  predicted_prob_col <- avoid_conflict(colnames(data), "predicted_probability")
 
   if (is.null(newdata)) {
     # use trained data and get probabilities
-    ret <- model$fitted.values %>%
+    f_values <- model$fitted.values
+
+    # f_values is one column matrix if the number of levels is 2
+    if(ncol(f_values) == 1){
+      f_values <- matrix(c(f_values, 1-f_values[,1]), ncol=2)
+      colnames(f_values) <- model$lev
+    }
+    ret <- f_values %>%
       as.data.frame() %>%
       append_colnames(prefix = "predicted_probability_")
 
+    # get max values from each row
+    p_values <- f_values[(max.col(f_values) - 1) * nrow(f_values) + seq(nrow(f_values))]
+    ret[[predicted_prob_col]] <- p_values
```

Comment:
Is p_value and predicted probability same thing? I can't quite convince myself on it.